src/lab.c

```diff
@@ -1231,6 +1231,12 @@ void CPUThink(GOBJ *event, GOBJ *hmn, GOBJ *cpu)
     // noact
     cpu_data->cpu.ai = 15;
 
+    if ((cpu_data->TM.state_prev[0] == ASID_CAPTURECUT && cpu_state != ASID_CAPTURECUT) || (cpu_data->TM.state_prev[0] == ASID_CAPTUREJUMP && cpu_state != ASID_CAPTUREJUMP))
```

Comment:
Please clean up this line, either by splitting it across multiple lines, or by extracting the conditions out of the if statement.